packages/applications/ssr/src/components/atoms/form/UploadDocument.tsx

```diff
@@ -11,6 +11,10 @@ export type UploadDocumentProps = {
   className?: string;
   label: React.ReactNode;
   name: string;
+  /*
+  Si utilisé dans un formulaire
+  l'id ne doit pas comprendre d'accent
+  */
```

Comment:
bonne idée de documenter ca, mais tant qu'à faire autant mettre ca en JSDoc valide, donc avec deux **
```suggestion
  /**
    Si utilisé dans un formulaire, l'id ne doit pas comprendre d'accent
  */
```